app/assets/javascripts/incidents/batch_export.coffee

```diff
@@ -3,49 +3,49 @@ alreadyExportedWarning = (count) ->
     ['incident has', 'this incident']
   else ['incidents have', 'these incidents']
   """
-  #{count} selected #{words[0]} already been exported.
+  #{count} selected #{words[0]} already been exported to Hastus.
   Please confirm that you would like to re-export #{words[1]}.
   """
 
 handleExporting = ->
-  inSelectMode = $('table.incidents th.batch-export').is ':visible'
+  inSelectMode = $('table.incidents th.batch-hastus-export').is ':visible'
```

Comment:
I didn't really *have* to change the CSS classes to differentiate them visually, but I also wanted to avoid any future programmer confusion (I get confused pretty easily).